src/CMakeLists.txt

```diff
@@ -203,7 +203,7 @@ if(DNNL_GPU_RUNTIME STREQUAL "OCL")
     install(FILES
         "../cmake/FindOpenCL.cmake"
         DESTINATION ${LIB_CONFIG_INSTALL_DIR})
-    install(FILES "${CMAKE_INSTALL_PREFIX}/${LIB_CONFIG_INSTALL_DIR}/FindOpenCL.cmake"
```

Comment:
Can you please clarify why this change is needed? Is it because there is an extra prefix?

Reply:
Yes looks related. If you pass a DESTDIR to cmake_install.cmake, then lets say you are installing to: 
/usr/lib/cmake/ and DESTDIR is D, FindOpenCL.cmake actually went to D//usr/lib/cmake/. So the next step will fail when copying file from /usr/lib/cmake.

Reply:
I had tried to include comments in the commit message. Perhaps should have copied them in pull request too ...

Reply:
Thanks. I should have read the commit messages. How standard is `DEST`? Do you know if other packaging tools use it too? I would probably prefer a mechanism that all the packagers could use.

Reply:
side note -- overall this change looks reasonable. why would we copy: `source->dst1, dst1->dst2` instead of `source->dst1,source->dst2`? so I am oK with the change too.

Reply:
> why would we copy: source->dst1, dst1->dst2 instead of source->dst1,source->dst2? 

Guilty. I actually thought that it would be simpler this way. I'm also OK with this change. I need to test if `DESTDIR` is supported by non-Makefile generators.